### PR TITLE
Show the config file path in the Preferences dialog

### LINKED ISSUE #325: [Enhancement Request] Add path to configuration file in ReText "About" window?
Hi,

[This link](https://github.com/retext-project/retext/blob/master/configuration.md) states:

> ReText stores all of its configuration in a text file. A path to that file is printed to stdout during ReText startup.

but if I'm on Windows and launching retext from the included batch file (retext.bat), it doesn't send anything to stdout. I did eventually find it at: "%appdata%\ReText project\ReText.ini"

For a cross-platform solution and to make it easier to find, could you add it to the About window?

### PR DESCRIPTION
**Pull request by xgouchet**
Fixes #325

### CODE REVIEW

File: ReText/window.py [hunk 1]
```diff
@@ -1166,7 +1166,8 @@ def aboutDialog(self):
 		+'</a> | <a href="http://daringfireball.net/projects/markdown/syntax">'
 		+self.tr('Markdown syntax')
 		+'</a> | <a href="http://docutils.sourceforge.net/docs/user/rst/quickref.html">'
-		+self.tr('reStructuredText syntax')+'</a></p>')
+		+self.tr('reStructuredText syntax')+'</a></p><p>'
+		+self.tr('Using configuration file at :')+'<br/>'+getSettingsFilePath()+'</p>')
```
Comment (mitya57) [inline ReText/window.py]: No space before `:` please, it should be there only in French :)

Comment (xgouchet) [reply ReText/window.py]: fixed

**Comment by xgouchet** [comment]
So I moved the information in the Config dialog, with a link to open the file as you suggested

**Comment by mitya57** [comment]
Thank you!

**Comment by mitya57** [comment]
I have redesigned the preferences dialog a bit, the link is now on top. Any feedback is welcome :)

**Comment by xgouchet** [comment]
This new tabbed interface looks great ! Good work Dmitry ! :+1:

**Pull request merged**